service/controllers/messages.go

```diff
@@ -399,3 +421,60 @@ func processBingResponses(result string) []messageQueryBody {
 	return queryResultArray
 }
 
+// processes youtube query result, scraps the required data and returns it
+func processYoutubeResponses(result string) []messageQueryBody {
+
+	subsl := "<a id=\"video-title\""
+	subsl2 := "href=\""
+	subsl3 := "</a>"
+	lensubsl3 := len(subsl3)
+
+	var queryResult messageQueryBody
+	var queryResultArray []messageQueryBody
+	var mid int
+
+	for i := 0; i < len(result) - len(subsl); i++ {
+		mess := ""
+		if result[i : i + len(subsl)] == subsl {
+			fmt.Println("reached1")
```

Comment:
if the code is working as expected, could you please remove these print statements

Reply:
Yaa sure.... sorry for the mistake